--- /zhome/cb/4/122206/torch/lib/python3.6/site-packages/torch/nn/modules/container.py
+++ /zhome/cb/4/122206/torch/lib/python3.6/site-packages/torch/nn/modules/container.py
@@ -45,11 +45,6 @@
     def __iadd__(self, modules):
         return self.extend(modules)
 
-    def __dir__(self):
-        keys = super(ModuleList, self).__dir__()
-        keys = [key for key in keys if not key.isdigit()]
-        return keys
-
     def append(self, module):
         r"""Appends a given module at the end of the list.
 